core/src/main/java/org/jdbi/v3/core/argument/ObjectPropertyNamedArgumentFinder.java

```diff
@@ -41,6 +42,7 @@
     }
 
     @Override
+    @SuppressWarnings("PMD.AvoidDeeplyNestedIfStmts")
```

Comment:
Is this PMD Supression ok, should it be for the line or should i remove the if statements in any way?